spaceallocator/spaceallocator.go

```diff
@@ -474,10 +528,12 @@ func (part *partition) removeOutdatedItems(requiredSize uint64) (freedSize uint6
 			"mountPoint": part.mountPoint, "id": item.id, "size": item.size,
 		}).Debug("Remove outdated item")
 
-		if err = item.allocator.removeOutdatedItem(item); err != nil {
+		if err := item.allocator.remover(item.id); err != nil {
```

Comment:
maybe remover should makes free space ?

Reply:
It was initial idea, but to do not remove not necessary items, we decided to pass size on adding outdated items. As we know the size, spaceallocator can free required size on item remove. So, no additional actions in remove callback are required.